src/nanoserver/1809/helix/amd64/Dockerfile

```diff
@@ -16,18 +16,8 @@ RUN curl -SL --output %TEMP%\python.zip https://www.nuget.org/api/v2/package/pyt
 ENV PATH="$PATH;C:\Program Files\PowerShell\;C:\Python;C:\python\scripts"
 
 RUN python -m pip install --upgrade pip `
-    && python -m pip install applicationinsights==0.11.8 `
-                             certifi==2019.3.9 `
-                             docker==3.7.2 `
-                             ndg-httpsclient==0.5.1 `
-                             psutil==5.6.1 `
-                             pyasn1==0.4.5 `
-                             pyopenssl==19.0.0 `
-                             requests==2.21.0 `
-                             six==1.12.0 `
-                             virtualenv==16.5.0 `
-                             vsts==0.1.20 `
-                             future==0.17.1 `
+    && python -m pip install virtualenv==16.5.0 &&
```

Comment:
Missing a ` character at the end of the line to continue to line.  Also suggest moving `&&` from the end of the line to the beginning of the next line.